e2e/interchaintestv8/operator/proofs.go

```diff
@@ -1,5 +1,13 @@
 package operator
```

Comment:
Does it make sense for the proof type to be under the operator at all? It's not an operator only thing, after all

Reply:
done